src/Core/src/Handlers/ActivityIndicator/ActivityIndicatorHandler.iOS.cs

```diff
@@ -1,17 +1,23 @@
-﻿using CoreGraphics;
+﻿using System;
+using CoreGraphics;
 using ObjCRuntime;
 using UIKit;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class ActivityIndicatorHandler : ViewHandler<IActivityIndicator, MauiActivityIndicator>
 	{
-		protected override MauiActivityIndicator CreatePlatformView() => new MauiActivityIndicator(CGRect.Empty, VirtualView)
+		protected override MauiActivityIndicator CreatePlatformView()
 		{
-#pragma warning disable CA1416 // TODO: 'UIActivityIndicatorViewStyle.Gray' is unsupported on: 'ios' 13.0 and later
-			ActivityIndicatorViewStyle = UIActivityIndicatorViewStyle.Gray
-#pragma warning restore CA1416
-		};
+			MauiActivityIndicator platformView;
+
+			if (OperatingSystem.IsIOSVersionAtLeast(13))
```

Comment:
This changes are not exactly related with the fix but, ported the code from Xamarin.Forms.